cms/djangoapps/models/settings/course_details.py

```diff
@@ -161,7 +165,8 @@ def update_from_json(cls, course_key, jsondict, user):
         # NOTE: below auto writes to the db w/o verifying that any of the fields actually changed
         # to make faster, could compare against db or could have client send over a list of which fields changed.
         for attribute in ABOUT_ATTRIBUTES:
-            cls.update_about_item(course_key, attribute, jsondict[attribute], descriptor, user)
+            if attribute in jsondict:
```

Comment:
@andy-armstrong -- just want to make sure this particular issue does not fall thru the cracks -- if you can take a look and let me know what you think, that would be great.